tests/python_functional/conftest.py

```diff
@@ -41,7 +41,7 @@
 # Command line options
 def pytest_addoption(parser):
     parser.addoption("--runslow", action="store_true", default=False, help="run slow tests")
-    parser.addoption("--run-with-valgrind", action="store_true", default=False, help="Run tests behind valgrind")
+    parser.addoption("--run-under", help="Run syslog-ng under selected tool, eg: [valgrind]")
```

Comment:
> eg: [valgrind]

I would change this to: `supported tools: valgrind`, and then add `strace` as well in the next commit.

Reply:
Thanks for the review note. I have fixed this.